build/import/HostAgnostic.props

```diff
@@ -30,11 +30,12 @@
     <PackageReference Include="Microsoft.DiaSymReader.Pdb2Pdb"      ExcludeAssets="all" />
     
     <!-- CPS -->
-    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem" />
```

Comment:
Do we no longer need the `.SDK` package?

Reply:
Having these packages is the equivalent to it but hope is to bring it back. Simply waiting for that package to be updated. For now using what's available